code/modules/mob/living/carbon/human/species_types/snail.dm

```diff
@@ -0,0 +1,10 @@
+/datum/species/snail
```

Comment:
undocumented species addition :)

Reply:
What the actual shit. This wasn’t supposed to pushed, I was saving it for another PR

Reply:
I was freaking out because someone said someone was adding snail people, but they were talking about me that forgot to remove this.